tests/registration_test.py

```diff
@@ -533,6 +533,52 @@ def test_strings_groupby_attach(self):
         self.assertIsInstance(sAttach.segments, ak.pdarray)
         self.assertIsInstance(sAttach.unique_keys, ak.Strings)
 
+    def test_string_property_accessor(self):
```

Comment:
it might be worth adding a test where u attach to make sure that functionality is working correctly

Reply:
I would appreciate you taking a look at the testing I added to see if this covers all the cases we need.